editor/import/resource_importer_scene.cpp

```diff
@@ -1549,7 +1549,7 @@ Node *EditorSceneImporterESCN::import_scene(const String &p_path, uint32_t p_fla
 
 	Error error;
 	Ref<PackedScene> ps = ResourceFormatLoaderText::singleton->load(p_path, p_path, &error);
-	ERR_FAIL_COND_V(!ps.is_valid(), NULL);
+	ERR_FAIL_COND_V_MSG(!ps.is_valid(), NULL, "Cannot load");
```

Comment:
Missing dot. Maybe missing text too :P "Cannot load scene as a text resource." or something like that.